lib/manageiq/providers/openstack/legacy/test/event_monitor_example.rb

```diff
@@ -0,0 +1,58 @@
+###################################################################################
+# Usage:
+# bundle exec rails r gems/pending/openstack/test/event_monitor_example.rb
+
+require 'manageiq-gems-pending'
```

Comment:
Do you need this?  It should already be required [here](https://github.com/ManageIQ/manageiq/blob/master/Gemfile#L9)